crowbar_framework/app/assets/stylesheets/content/footer.scss

```diff
@@ -1,8 +1,10 @@
 footer {
-  color: #999;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
   font-size: 12px;
-  padding: 0 20px 20px 20px;
   text-align: center;
+  color: #999;
```

Comment:
Color literals like `#999` should only be used in variable declarations; they should be referred to via variable everywhere else.